modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/senna/screen/HtmlScreen.js

```diff
@@ -345,28 +343,14 @@ class HtmlScreen extends RequestScreen {
 		});
 	}
 
-	/**
-	 * Creates a new element from given, copies attributes, mutates href to be
-	 * unique to prevent caching and more than one load/error event from firing.
-	 */
-	replaceStyleAndMakeUnique_(style) {
```

Comment:
This wasn't used anymore. I think it was to avoid caching in IE11, so... good riddance!